x-pack/plugins/lists/server/services/exception_lists/get_exception_list_summary.test.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ExceptionListSummarySchema } from '@kbn/securitysolution-io-ts-list-types';
+
+import type { SavedObjectsClientContract } from '../../../../../../src/core/server';
+import { savedObjectsClientMock } from '../../../../../../src/core/server/mocks';
+
+import { getExceptionListSummary } from './get_exception_list_summary';
+
+describe('get_exception_list_summary', () => {
+  describe('getExceptionListSummary', () => {
+    let savedObjectsClient: jest.Mocked<SavedObjectsClientContract>;
+
+    beforeEach(() => {
+      savedObjectsClient = savedObjectsClientMock.create();
+    });
+
+    test('it should aggregate items if not host isolation exception artifact', async () => {
+      const savedObject = {
+        aggregations: {
+          by_os: {
+            buckets: [
+              { doc_count: 2, key: 'linux' },
+              { doc_count: 3, key: 'macos' },
+              { doc_count: 5, key: 'windows' },
+            ],
+            doc_count_error_upper_bound: 0,
+            sum_other_doc_count: 0,
+          },
+        },
+        page: 1,
+        per_page: 0,
+        saved_objects: [],
+        total: 7,
+      };
+      savedObjectsClient.find.mockResolvedValue(savedObject);
+
+      const summary = (await getExceptionListSummary({
+        filter: undefined,
+        id: undefined,
+        listId: '',
+        namespaceType: 'agnostic',
+        savedObjectsClient,
+      })) as ExceptionListSummarySchema;
+
+      expect(summary.total).toEqual(10);
+    });
+
+    test('it should NOT aggregate items if host isolation exception artifact', async () => {
+      const savedObject = {
+        aggregations: {
+          by_os: {
+            buckets: [
+              { doc_count: 3, key: 'linux' },
+              { doc_count: 3, key: 'macos' },
+              { doc_count: 3, key: 'windows' },
+            ],
+            doc_count_error_upper_bound: 0,
+            sum_other_doc_count: 0,
+          },
+        },
+        page: 1,
+        per_page: 0,
+        saved_objects: [],
+        total: 7,
```

Comment:
Not relevant but I think this total number should be 6 according the counters above

Reply:
@dasansol92 so this was because the API was returning +1 as total. I went looking for why that was and @paul-tavares helped me figure that out. We made the changes in bdc0c97f1f7835f7560e6f725d7663800ac016f2 to fix that. Also updated the test mock to have the correct total.